BUILD

```diff
@@ -1,13 +1,37 @@
 package(default_visibility = ["//visibility:public"])
 
-ALL_MODULES = glob(["asmcli/**/*.sh"], exclude = ["**/test*/**"],)
+ASMCLI_MODULES = glob(["asmcli/**"])
+NODE_MODULES = glob(["node_modules/**"])
+ASMCLI_SOURCES = glob(["asmcli/**/*.sh"], exclude = ["**/test*/**"],)
+ALL_UNIT_TESTS = glob(["asmcli/tests/*.bats"])
```

Comment:
Let's just say TESTS in case we decide to put some non unit ones in there.

Reply:
Done